koku/masu/database/presto_sql/reporting_ocpazurecostlineitem_daily_summary.sql

```diff
@@ -124,133 +251,240 @@ cte_ocp_on_azure_tag_joined AS (
+    cluster_id,
+    cluster_alias,
+    data_source,
+    namespace,
+    node,
+    persistentvolumeclaim,
+    persistentvolume,
+    storageclass,
+    pod_labels,
+    resource_id,
+    usage_start,
+    usage_end,
+    cost_entry_bill_id,
+    subscription_guid,
+    instance_type,
+    service_name,
+    resource_location,
+    usage_quantity,
+    unit_of_measure,
+    currency,
+    pretax_cost,
+    markup_cost,
+    project_markup_cost,
+    pod_cost,
+    tags,
+    source_uuid
+)
+SELECT uuid(),
+    {{report_period_id | sqlsafe}} as report_period_id,
+    cluster_id as cluster_id,
+    cluster_alias as cluster_alias,
+    data_source,
+    namespace,
+    node,
+    persistentvolumeclaim,
+    persistentvolume,
+    storageclass,
+    json_parse(pod_labels),
+    resource_id,
+    date(usage_start),
+    date(usage_end),
     {{bill_id | sqlsafe}} as cost_entry_bill_id,
-    ocp_azure.subscription_guid,
-    ocp_azure.instance_type,
-    ocp_azure.service_name,
-    ocp_azure.resource_location,
-    ocp_azure.usage_quantity / pc.project_count / dsc.data_source_count as usage_quantity,
-    ocp_azure.pretax_cost / pc.project_count / dsc.data_source_count as pretax_cost,
-    ocp_azure.pretax_cost / pc.project_count / dsc.data_source_count * cast({{markup}} as decimal(24,9)) as markup_cost,
-    ocp_azure.currency,
-    ocp_azure.unit_of_measure,
-    CASE WHEN ocp_azure.resource_id_matched = TRUE AND ocp_azure.data_source = 'Pod'
-        THEN (ocp_azure.{{pod_column | sqlsafe}} / ocp_azure.{{cluster_column | sqlsafe}}) * ocp_azure.pretax_cost / dsc.data_source_count
-        ELSE ocp_azure.pretax_cost / pc.project_count / dsc.data_source_count
-    END as pod_cost,
-    CASE WHEN ocp_azure.resource_id_matched = TRUE AND ocp_azure.data_source = 'Pod'
-        THEN (ocp_azure.{{pod_column | sqlsafe}} / ocp_azure.{{cluster_column | sqlsafe}}) * ocp_azure.pretax_cost * cast({{markup}} as decimal(24,9)) / dsc.data_source_count
-        ELSE ocp_azure.pretax_cost / pc.project_count / dsc.data_source_count * cast({{markup}} as decimal(24,9))
-    END as project_markup_cost,
-    json_parse(ocp_azure.tags) as tags,
-    UUID '{{azure_source_uuid | sqlsafe}}' as source_uuid
-FROM cte_ocp_on_azure_joined AS ocp_azure
-JOIN cte_project_counts AS pc
-    ON ocp_azure.azure_id = pc.azure_id
-JOIN cte_data_source_counts AS dsc
-    ON ocp_azure.azure_id = dsc.azure_id
-        AND ocp_azure.namespace = dsc.namespace
+    subscription_guid,
+    instance_type,
+    service_name,
+    resource_location,
+    usage_quantity,
+    unit_of_measure,
+    currency,
+    pretax_cost,
+    markup_cost,
+    project_markup_cost,
+    pod_cost,
+    json_parse(tags),
+    cast(azure_source as UUID)
+FROM hive.{{schema | sqlsafe}}.reporting_ocpazurecostlineitem_project_daily_summary
+WHERE azure_source = '{{azure_source_uuid | sqlsafe}}'
+    AND ocp_source = '{{ocp_source_uuid | sqlsafe}}'
+    AND year = {{year}}
+    AND month = {{month}}
```

Comment:
We may run into problems here with single digit months like we did with AWS, we had to pad the numbers to make sure the dates would line up, [PR1](https://github.com/project-koku/koku/pull/3362) --- [PR2](https://github.com/project-koku/koku/pull/3380)

Reply:
Thanks for pointing these out @cgoodfred I will make sure to update them today. 👍

Reply:
Use this one: https://github.com/project-koku/koku/pull/3386 instead of 3383 (hive table duplication), there was still an issue

Reply:
Ah, thanks @cgoodfred ill update this now

Reply:
Updated! Thanks again for keeping me in the loop about these changes!